Components/Hlms/Pbs/include/Cubemaps/OgreCubemapProbe.h

```diff
@@ -47,6 +47,14 @@ namespace Ogre
         friend class ParallaxCorrectedCubemap;
         friend class ParallaxCorrectedCubemapAuto;
 
+    public:
+        enum TimeSlicing
+        {
+            TS_ONE_FRAME = 6,
+            TS_SIX_FRAMES = 1
+        };
```

Comment:
Other values can be added like `TS_TWO_FRAMES = 3` and `TS_THREE_FRAMES = 2`

Reply:
Three possible behaviours are supported at the moment:

- `TS_NONE`: no time slicing at all (cubemap generation process is done in a single frame)
- `TS_ALL_FACES`: all cubemap faces are rendered at the same frame, while mipmaps generation and ibl_specular pass happen at the subsequent frame (cubemap generation process takes two frames)
- `TS_INDIVIDUAL_FACE`: like above, but a single cubemap face is rendered per frame (cubemap generation process takes seven frames)